crc-bundle-info.json.sample

```diff
@@ -7,12 +7,15 @@
   # - 1.1: addition of 'name'
   # - 1.2: addition of 'storage.fileList'
   # - 1.3: remove of 'clusterInfo.kubeadminPasswordFile'
-  "version": "1.3",
+  # - 1.4: addition of 'arch'
+  "version": "1.4",
   # Type of this bundle content
   # Currently the only valid type is 'snc' (which stands for 'single-node-cluster')
   "type": "snc",
   # Name of the bundle
   "name": "crc_libvirt_4.6.1",
+  # Bundle arch
```

Comment:
What are the valid values? This follows https://gist.github.com/lizkes/975ab2d1b5f9d5fdee5d3fa665bcfde6 with amd64/arm64 being used at the moment.

Reply:
With the previous question, I meant the valid values should be documented in the sample file.

Reply:
Ack